index.bs

```diff
@@ -1543,6 +1585,32 @@ the following steps must be followed:
   <a>Terminate this algorithm</a>.
 4. If no error has occurred, return the <var>result</var> of the <a>read operation</a> as an {{ArrayBuffer}}.
 
+<h5 id="readAsBinaryStringSyncSection">
+The {{FileReaderSync/readAsBinaryString()}} method</h5>
+
+When the <dfn method for=FileReaderSync id="dfn-readAsBinaryStringSync">readAsBinaryString(blob)</dfn> method is called,
+the following steps must be followed:
+
+1. If {{FileReader/readyState}} = {{FileReader/LOADING}}
+  throw an {{InvalidStateError}} exception
+  and <a>terminate this algorithm</a>.
+2. If the <code>blob</code> has been closed through the {{Blob/close()}} method,
+  throw an {{InvalidStateError}} exception
+  and <a>terminate this algorithm</a>.
+3. Otherwise, initiate a <a>read operation</a> using the <code>blob</code> argument,
+  and with the <a>synchronous flag</a> <em>set</em>.
+  If the <a>read operation</a> returns failure,
+  throw the appropriate exception as defined in [[#dfn-error-codes]].
+  <a>Terminate this algorithm</a>.
+4. If no error has occurred, return the <var>result</var> of the
+  <a>read operation</a> as an <a>binary string</a>.
```

Comment:
Same here. I actually thought "binary string" was an argument to "read operation" when reading this initially. A comma would help I think, though it's still not great that "binary string" is defined in some non-normative description of a method.